examples/kafkatokafka/parser.go

```diff
@@ -0,0 +1,47 @@
+package kafkatokafka
+
+import (
+	"errors"
+
+	"github.com/digitalocean/captainslog"
+	"github.com/digitalocean/firebolt"
+	"github.com/digitalocean/firebolt/fbcontext"
+	log "github.com/sirupsen/logrus"
+)
+
+// Parser is a firebolt node for parsing syslog messages from bytes using `digitalocean/captainslog`.
```

Comment:
I know it literally says it's a firebolt node, but I think it'd be cool to make it 100% clear what it's implementing so people could connect the dots a bit more.

Reply:
are you suggesting changing the comment to ```// Parser is a firebolt `node.SyncNode` for...```?

Reply:
I'd be like `Parser is a firebolt `node.SyncNode` for .... SyncNodes do x and y, this is why it is the right choice for doing this`. Basically overexplaining everything so that if this was the only thing I saw of firebolt I'd be able to mostly understand what it was all for. Again totally my biased opinion, in my experience a lot people will click the repo, read the first line of the readme, and go straight to an example and if it looks hard and doesn't connect with that one line explanation, click out and leave forever.

Reply:
got it, yep that's great i'll do that too

Reply:
done